src/native/corehost/hostpolicy/deps_format.cpp

```diff
@@ -214,25 +214,34 @@ namespace
     };
 
     // Returns the RID determined (computed or fallback) for the platform the host is running on.
```

Comment:
We should probably update the comment to make it very clear that this intentionally returns the computed RID, so possibly distro specific with version number, as oppose to everywhere else where we use the host rid (the one the host was built with).

Maybe we should actually rename this method to something like `get_current_machine_rid` or something similar, to make it very explicit that this RID comes from the OS, not from the host (and thus requires full RID graph).